plugins/reporters/spdx/src/main/kotlin/Extensions.kt

```diff
@@ -0,0 +1,171 @@
+/*
```

Comment:
Now, as we have move to such a fine grained module structure - to me it makes even less sense to determine a location for a function based on whether it's an extension function or not and then as second priority the alphabetical order. 

IMO cohesion is more important: e.g. if a non-extension utility function is closely related to an extension function, then IMO it should be located nearby.

What do you think? In particular about the alternatives to

1. Keep the functions where they currently are
2. Have a utils class where when can also put non-extension functions.

Reply:
> to me it makes even less sense to determine a location for a function based on whether it's an extension function or not [...]  if a non-extension utility function is closely related to an extension function, then IMO it should be located nearby

I'm open to the idea to group functions, extensions or not, by purpose instead of syntax, although having an `Extensions.kt` seems to be pretty common in Kotlin projects. Anyway, I guess we should do this consistently throughout the code base in a separate PR then.